build/images/ovs/Dockerfile.windows

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG NANOSERVER_VERSION
+
+FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as windows-ovs
+
+ARG WIN_OVS_VERSION
+
+# Switch to a privileged user
+USER ContainerAdministrator
+
+# Sets the shell to be PowerShell 
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ENV WIN_OVS_VERSION ${WIN_OVS_VERSION}
+
+# Install VC for ovs
+RUN curl.exe -Lo C:\vc_redist.x64.exe https://aka.ms/vs/17/release/vc_redist.x64.exe; \
+    Start-Process -FilePath C:\vc_redist.x64.exe -ArgumentList ‘/install’,’/quiet’,’/norestart’ -Verb RunAs -Wait; \
+    rm C:\vc_redist.x64.exe
+
+# Download and extract OVS
+RUN curl.exe -fLo C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip https://downloads.antrea.io/ovs/ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip; \
+    Expand-Archive C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip -DestinationPath C:\  ; \
+    rm C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip
+
+# Download, extract OpenSSL and copy libraries
+RUN curl.exe -Lo C:\openssl-1.0.2u-x64_86-win64.zip https://indy.fulgan.com/SSL/openssl-1.0.2u-x64_86-win64.zip; \
+    Expand-Archive C:\openssl-1.0.2u-x64_86-win64.zip -DestinationPath C:\openssl  ;  \
+    Move-Item -Path 'C:\openssl\libeay32.dll', 'C:\openssl\ssleay32.dll' -Destination 'C:\Windows\System32\' ; \
```

Comment:
This command may fail with error "Move-Item : A positional parameter cannot be found that accepts argument 'C:\openssl\ssleay32.dll'" ? powershell command "Move-Item" seems not support to move two files to a single target ?

Reply:
BTW, there is no "DestinationPath" in Move-Item command, https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/move-item?view=powershell-7.3

Reply:
@wenyingd 
-Destination is 'C:\Windows\System32\' , it works fine, able to move two files to C:\Windows\System32\

Reply:
Please also update the commit message.

Reply:
PS C:\Windows\System32> ls -l .\libeay32.dll


    Directory: C:\Windows\System32


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----       12/21/2019  12:01 PM        2293248 libeay32.dll


PS C:\Windows\System32> ls -l .\ssleay32.dll


    Directory: C:\Windows\System32


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----       12/21/2019  12:01 PM         386560 ssleay32.dll

Reply:
OK, I miss the "," between the two files.

Reply:
> Please also update the commit message.

commit message updated